Carmu/Sources/Controllers/MyPage/MyPageViewController.swift

```diff
@@ -25,7 +31,8 @@ final class MyPageViewController: UIViewController {
                 return
             }
             self.myPageView.nicknameLabel.text = userData.nickname
-            self.myPageView.imageView.image = UIImage(named: userData.profileType.rawValue)
+            self.selectedProfileType = userData.profileType
+            self.updateProfileImageView(profileType: self.selectedProfileType)
```

Comment:
`self.` 키워드는 일반적으로 꼭 필요한 경우에만 사용하는 것 같아요 😊

```suggestion
            myPageView.nicknameLabel.text = userData.nickname
            selectedProfileType = userData.profileType
            updateProfileImageView(profileType: self.selectedProfileType)
```

Reply:
이 코드는 클로저 내부라서 `self`를 생략할 수가 없더라구요🥲